chainstate/test-suite/src/tests/nft_utils.rs

```diff
@@ -0,0 +1,22 @@
+// Copyright (c) 2022 RBB S.r.l
+// opensource@mintlayer.org
+// SPDX-License-Identifier: MIT
+// Licensed under the MIT License;
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// https://github.com/mintlayer/mintlayer-core/blob/master/LICENSE
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use common::chain::tokens::TokenCreator;
+use crypto::key::{KeyKind, PrivateKey};
+
+pub fn random_creator() -> Option<TokenCreator> {
```

Comment:
Why is the returned value Option?

Reply:
As we spoke, it was an abstraction problem. 
I fixed it.